src/storage/src/source/generator/marketing.rs

```diff
@@ -176,14 +175,23 @@ impl Generator for Marketing {
                     }
 
                     if converted {
-                        let converted_at = now() + rng.gen_range(1..30);
+                        let converted_at = now() + rng.gen_range(5..30);
 
-                        future_updates.insert(converted_at, (LEADS_OUTPUT, lead.to_row(), -1));
```

Comment:
github doesnt let me add a comment there, but youll need to fix the comment on the `impl` block above, as its now monotonic